Source/EmptyContent/EmptyContent.swift

```diff
@@ -23,15 +23,22 @@ import UIKit
 
 //Represents content which gets displayed during empty states.
 public struct EmptyContent {
-    public init(title: String? = nil, subtitle: String? = nil, image: UIImage? = nil) {
+    public init(title: String? = nil, subtitle: String? = nil, image: UIImage? = nil, shouldAllowTouch: Bool? = false, shouldAllowScroll: Bool? = false, shouldAllowImageViewAnimate: Bool? = false) {
```

Comment:
It does not make sense to let the content decide if it is scrollable or not. Content contains text am images, no other attributes.

We should find a better place to locate these attributes